src/web/routes/application/are-you-pregnant/are-you-pregnant.js

```diff
@@ -8,7 +8,7 @@ const pageContent = ({ translate }) => ({
   yes: translate('yes'),
   no: translate('no'),
   expectedDeliveryDateText: translate('areYouPregnant.expectedDeliveryDateText'),
-  expectedDeliveryDateHint: translate('areYouPregnant.expectedDeliveryDateHint')
+  expectedDeliveryDateHint: translate('areYouPregnant.expectedDeliveryDateHint', { exampleDate: exampleDate() })
```

Comment:
It's probably best to define `exampleDate()` before its used. In some situations this could lead to a reference error, see https://eslint.org/docs/rules/no-use-before-define